<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelDescription">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="buddy">
         <cstring>textEditDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="textEditDescription"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOwner">
        <property name="text">
         <string>Owner:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditOwner</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditOwner"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelAims">
        <property name="text">
         <string>Aims:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAims</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditAims"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelContext">
        <property name="text">
         <string>Context:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditContext</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditContext"/>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="labelMetadata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Metadata</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
        <property name="margin">
         <number>-3</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelFilePath">
        <property name="text">
         <string>File path:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditFilePath</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditFilePath">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelFileName">
        <property name="text">
         <string>File name:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditFileName</cstring>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="lineEditFileName">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="labelCreationDate">
        <property name="text">
         <string>Creation date:</string>
        </property>
        <property name="buddy">
         <cstring>dateTimeEditCreationDate</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditCreationDate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelModificationDate">
        <property name="text">
         <string>Modification date:</string>
        </property>
        <property name="buddy">
         <cstring>dateTimeEditModificationDate</cstring>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEditModificationDate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
